integration-cli/docker_cli_run_unix_test.go

```diff
@@ -238,7 +238,7 @@ func (s *DockerSuite) TestRunOOMExitCode(c *check.C) {
 	select {
 	case err := <-errChan:
 		c.Assert(err, check.IsNil)
-	case <-time.After(30 * time.Second):
+	case <-time.After(600 * time.Second):
```

Comment:
@brahmaroutu Any idea why this case might takes so long? I found on our CI server, this case takes almost 400s, though it's passed, and it's a x86-64 server.